README.md

```diff
@@ -1,160 +1,183 @@
+![pnpm](https://img.shields.io/static/v1?style=for-the-badge&message=pnpm&color=222222&logo=pnpm&logoColor=F69220&label=)
 
-```js
-import Worker from 'whatwg-worker'
-
-const worker = new Worker('data:text/javascript,postMessage("hello")')
-worker.onmessage = e => console.log(e.data)  // "hello"
-```
-
-<table>
-<thead><tr><th><strong>main.js</strong></th><th><strong>worker.js</strong></th></tr></thead>
-<tbody><tr><td>
-
-```js
-import Worker from 'whatwg-worker'
-
-const url = new URL('./worker.js', import.meta.url)
-const worker = new Worker(url)
-
-worker.addEventListener('message', e => {
-  console.log(e.data)  // "hiya!"
-})
+Normally, you'll want to use this package in a Node.js environment. If you
+import it using an npm CDN like [ESM>CDN] or [jsDelivr], you'll just be getting
+the native global `Worker` class. You can install it locally for use in Node.js
+using npm, [Yarn], [pnpm], or your other favorite npm-related package manager.
 
-worker.postMessage('hello')
+```sh
+npm install whatwg-worker
 ```
 
-</td><td valign="top">
-
-```js
-addEventListener('message', e => {
-  if (e.data === 'hello') {
-    postMessage('hiya!')
-  }
-})
-```
-
-</td></tr></tbody>
-</table>
-
-👉 Notice how `new URL('./worker.js', import.meta.url)` is used above to load the worker relative to the current module instead of the application base URL. Without this, Worker URLs are relative to a document's URL, which in Node.js is interpreted to be `process.cwd()`.
-
-> _Support for this pattern in build tools and test frameworks is still limited. We are [working on growing this](https://github.com/developit/web-worker/issues/4)._
+## Usage
 
-### Module Workers
+![Node.js](https://img.shields.io/static/v1?style=for-the-badge&message=Node.js&color=339933&logo=Node.js&logoColor=FFFFFF&label=)
 
-Module Workers are supported in Node 12.8+ using this plugin, leveraging Node's native ES Modules support.
-In the browser, they can be used natively in Chrome 80+, or in all browsers via [worker-plugin] or [rollup-plugin-off-main-thread]. As with classic workers, there is no difference in usage between Node and the browser:
+The easiest way to use this package is just to import it as a polyfill. This
+will add the `Worker` class to the global scope just like in the browser.
 
-<table>
-<thead><tr><th><strong>main.js</strong></th><th><strong>worker.js</strong></th></tr></thead>
-<tbody><tr><td>
+<table><td>
 
 ```js
-import Worker from 'whatwg-worker'
-
-const worker = new Worker(
-  new URL('./worker.js', import.meta.url),
-  { type: 'module' }
-)
-worker.addEventListener('message', e => {
-  console.log(e.data)  // "200 OK"
-})
-worker.postMessage('https://httpstat.us/200')
+// app.js
+import "whatwg-worker";
+
+// 💡 You can use 'new URL()' or 'import.meta.resolve()' to pass a URL relative
+// to the current file! Otherwise, it'd be relative to 'process.cwd()'.
+const worker = new Worker(new URL("worker.js", import.meta.url));
```

Comment:
As we are only supporting type module at the moment i think we should give a more correct example like this

```js
const worker = new Worker(
  new URL('./worker.js', import.meta.url),
  { type: 'module' }
)
```